src/components/Cart/Cart.js

```diff
@@ -1,49 +1,133 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import CartProductsListItem from '../CartProductsListItem/CartProductsListItem';
+import CartList from '../CartProductsListItem/CartProductsListItem';
 import './Cart.scss';
 
-const Cart = ({ isCartOpen, setIsCartOpen }) => {
+const Cart = ({ setIsCartOpen }) => {
+  const [cartData, setCartData] = useState([]);
   const navigate = useNavigate();
 
+  const clickOrder = () => {
+    navigate('/order');
+    setIsCartOpen(false);
+  };
+
+  useEffect(() => {
+    fetch('http://10.58.52.228:8002/carts')
+      .then(response => response.json())
+      .then(data => setCartData(data.carts));
+  }, []);
+
+  const handleItemNum = (value, quantity, cartId) => {
+    if (quantity + value === 0) return;
+
+    const newArr = [...cartData];
+
+    setCartData(
+      newArr.map(list => {
+        if (list.cartId === cartId) {
+          return { ...list, quantity: quantity + value };
+        }
+        return list;
+      })
+    );
+
+    changeQuantity(quantity + value, cartId);
+  };
+
+  const changeQuantity = (quantity, cartId) => {
+    fetch('http://10.58.52.228:8002/carts', {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('TOKEN'),
+      },
+      body: JSON.stringify({
+        cartId: cartId,
+        quantity: quantity,
+      }),
```

Comment:
```suggestion
      body: JSON.stringify({
        cartId
        quantity
      }),
```

JS에 단축속성명이라는 개념이 있습니다.
해당 개념을 적용하면 위와 같이 객체를 구현할 수 있습니다!

자세한 내용은 아래 참조드린 블로그 확인해주세요!
https://bloodstrawberry.tistory.com/404